Objects/longobject.c

```diff
@@ -6775,3 +6776,115 @@ int PyLong_AsUInt64(PyObject *obj, uint64_t *value)
 {
     LONG_TO_UINT(obj, value, "C uint64_t");
 }
+
+
+static const PyLongLayout PyLong_LAYOUT = {
+    .bits_per_digit = PyLong_SHIFT,
+    .digits_order = -1,  // least significant first
+    .endianness = PY_LITTLE_ENDIAN ? -1 : 1,
+    .digit_size = sizeof(digit),
+};
+
+
+const PyLongLayout* PyLong_GetNativeLayout(void)
```

Comment:
Nit

```suggestion
const PyLongLayout *PyLong_GetNativeLayout(void)
```

Reply:
I prefer to put the star aside the type name.